### PR TITLE
Google auth

### PR DESCRIPTION
**Pull request by bartoszmaka**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-          :rememberable, :validatable
+         :rememberable, :validatable,
+         :omniauthable, omniauth_providers: [:google_oauth2]
+
+  def self.from_omniauth(access_token)
+    data = access_token.info
+    user = User.where(email: data['email']).first
```
Comment (kovson) [inline app/models/user.rb]: Try to write it without ```.first```. Where will be substituted for another method.

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -13,6 +13,11 @@
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
   config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+                  hd: 'selleo.com'
+  # config.omniauth :google_oauth2, ENV["GOOGLE_OAUTH_CLIENT_ID"], ENV["GOOGLE_OAUTH_SECRET"], {
```
Comment (kovson) [inline config/initializers/devise.rb]: ✂️

**Comment by kovson** [comment]
please clean up commits

**Comment by synion** [comment]
@kovson All commits was repair i think now it's better

**Pull request merged**